integration-cli/docker_cli_save_load_unix_test.go

```diff
@@ -41,41 +40,27 @@ func (s *DockerSuite) TestSaveAndLoadRepoStdout(c *check.C) {
 	loadCmd := exec.Command(dockerBinary, "load")
 	loadCmd.Stdin = tmpFile
 
-	if out, _, err = runCommandWithOutput(loadCmd); err != nil {
-		c.Fatalf("failed to load repo: %s, %v", out, err)
-	}
+	out, _, err = runCommandWithOutput(loadCmd)
+	c.Assert(err, check.IsNil, check.Commentf(out))
 
 	after, _ := dockerCmd(c, "inspect", repoName)
 
-	if before != after {
-		c.Fatalf("inspect is not the same after a save / load")
-	}
+	c.Assert(before, check.Equals, after, check.Commentf("inspect is not the same after a save / load"))
 
 	deleteImages(repoName)
 
 	pty, tty, err := pty.Open()
-	if err != nil {
-		c.Fatalf("Could not open pty: %v", err)
-	}
+	c.Assert(err, check.IsNil)
 	cmd := exec.Command(dockerBinary, "save", repoName)
 	cmd.Stdin = tty
 	cmd.Stdout = tty
 	cmd.Stderr = tty
-	if err := cmd.Start(); err != nil {
-		c.Fatalf("start err: %v", err)
-	}
-	if err := cmd.Wait(); err == nil {
-		c.Fatal("did not break writing to a TTY")
-	}
+	c.Assert(cmd.Start(), check.IsNil)
+	c.Assert(cmd.Wait(), check.NotNil, check.Commentf("did not break writing to a TTY"))
```

Comment:
same
this commentf can be a standard comment on the same line as there are no variables being interpolated.